lang/en/tool_mfa.php

```diff
@@ -67,6 +67,8 @@
 $string['mfa'] = 'MFA';
 $string['mfasettings'] = 'Manage MFA';
 $string['na'] = 'n/a';
+$string['nologinusers'] = 'Not logged in';
+$string['nonauthusers'] = 'Logged in but not authenticated';
```

Comment:
This feels ambiguous but not sure what is better, thoughts:

"Logged in but not MFA'ed" <- what I mean but sucks

"Confirmed password but no other factors"

Reply:
'Logged in but not MFA authenticated' Feels best to me

Reply:
Going with Pending MFA